includes/class-rest.php

```
@@ -840,6 +876,194 @@ public function run_performance_scan( \WP_REST_Request $request ): \WP_REST_Resp
+
+			$deleted = Telemetry_Table::truncate();
+
+			// Clear all registered transient keys.
+			$transient_index = get_option( 'wppo_transient_index', array() );
+			foreach ( $transient_index as $key ) {
+				delete_transient( $key );
+			}
+			update_option( 'wppo_transient_index', array() );
+
+			return $this->send_response( array( 'deleted' => $deleted ) );
+		}
```

PHPMD (2.15.0)

[error] The method delete_telemetry is not named in camelCase. (undefined)

(CamelCaseMethodName)